x/monitoringp/client/cli/query_consumer_client_id.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/monitoringp/types"
+)
+
+func CmdShowConsumerClientID() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "show-consumer-client-id",
+		Short: "shows consumerClientID",
```

Comment:
```suggestion
		Short: "shows the consumerClientID",
```

Reply:
We can even put `shows the consumer client ID used for a chain IBC connection`